crypto/bio/bss_conn.c

```diff
@@ -8,7 +8,6 @@
  */
 
 #include <stdio.h>
-#include <errno.h>
```

Comment:
maybe undo this too.

Reply:
Done (though the #include is not really needed, 
but `get_last_socket_error()` indirectly uses `errno`).

Reply:
I wear special roentgen glasses that look throu macros :-)